frontend/src/lib/components/InsightCard/InsightCard.scss

```diff
@@ -0,0 +1,107 @@
+.InsightCard {
+    transition: border 200ms ease;
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+    height: 30rem;
+    border-radius: var(--radius);
```

Comment:
Nit: Prefer SCSS variables over CSS global vars

Reply:
I'm of a quite strong opinion that CSS vars should be preferred unless the value really must be baked in at compile time (e.g. due to some calculations using Sass functions). CSS vars have been introduced ~5 years ago to replace all the various preprocessor-based solutions like Sass, and now have great browser support. The advantages are:
- no dependency on a preprocessor
- inheritance working just like with regular inherited CSS properties
- there's ALWAYS a single source of truth, even when working with a fully built frontend – no need to switch back and forth between the browser and editor, just edit the relevant `--variable` in devtools

Reply:
Alright those seem like pretty solid arguments, let's use CSS then